install_config/configuring_gce.adoc

```diff
@@ -48,8 +48,15 @@ During xref:../install_config/install/advanced_install.adoc#install-config-insta
 
 [IMPORTANT]
 ====
-When using GCE, the `openshift_gcp_project` and `openshift_gcp_prefix`
+* When using GCE, the `openshift_gcp_project` and `openshift_gcp_prefix`
 parameters must be defined.
+* For running loadbalancer services using the Google compute platform, the nodes
```

Comment:
@gaurav-nelson From a quick search of the docs, I think s/loadbalancer/load balancer

Unless you're talking about the LoadBalancer service?